process-compose.yaml

```diff
@@ -332,6 +333,8 @@ processes:
       - ESPRESSO_SEQUENCER_IDENTITY_LATITUDE=-25.2744
       - ESPRESSO_SEQUENCER_IDENTITY_LONGITUDE=133.7751
       - ESPRESSO_SEQUENCER_PUBLIC_API_URL=http://localhost:$ESPRESSO_SEQUENCER4_API_PORT/
+      - ESPRESSO_SEQUENCER_SQLITE_PATH=$ESPRESSO_BASE_STORAGE_PATH/seq4-sqlite.db
```

Comment:
IMO we shouldn't have a separate variable for this. It should just be some fixed sub-directory of `ESPRESSO_SEQUENCER_STORAGE_PATH`, so that this is backwards compatible with the current FS-based storage (eventually we will ask operators to switch)